README.md

```diff
@@ -58,4 +103,61 @@ An example of a **meaningful** change:
 - `falsepos-...` files are cases any filter should catch
 - `truepos...` files are cases of changes we care about
 
-Filters will need to be tested (and possibly trained) against real datasets at some point
+This is a small but illustrative sample. Many more samples will be made
+available as soon as possible.
+
+### Schema
+
+This describes the schema of the SQL databases shared by the Rails app in
+web-monitoring-db and the Python processing bakend in web-monitoring-processing.
+Review the Definition of Terms section above, which corresponds to these tables.
+
+Every table includes:
+
+* uuid: UUID4 unique indentifier
+* created_at: internal detail of the database
+* updated_at: internal detail of the database
+
+in addition to the table-specific fields listed below.
+
+#### Pages
+
+* url: URL, which may be updated over time if a page is moved
+* title: `<title>` tag
+* agency: Governemnt agency
+* site: Goverment site (?)
+
```

Comment:
If I understand correctly, we plan to remove `Page.versionista_account` and stash that as part of `Version.source_metadata`, since it's Versionista-specific stuff that wouldn't belong on a Version captured from some other source.